### ISSUE TITLE
Tremor Totem

### ISSUE THREAD

**Issue by Bustyplz** [Atlantiss/BugTracker#5350]
Hi,
There is a little problem with the Tremor Totem, it doesn't work on 2 spell : 

-  Intimidating Shout (fear of the warrior, it doesn't work and its not normal)
- Seduction (the Tremor Totem work on the Charm and Sleep effect, it's write on the spell)

![wowscrnshot_122215_180335](https://cloud.githubusercontent.com/assets/16181538/11961109/ca3179ba-a8d6-11e5-8c3b-0cd54b413179.jpg)

You can see on this screen the bug for the Intimidating Shout , on the top of this screen its write "Can't do that while stunned" and its not a stunned but a fear 

Sorry i didn't find a warlock for the screen but it's the same problem with the Seduction of the Succubus (warlock pet)

**Comment by Kejmil** [comment]
main target of aoe warrior fear is in different effect than aoe targets , it is normal from what i recall

**Comment by Bustyplz** [comment]
no, its not normal because the Warlock have an Aoe fear and the tremor work on this fear and trust me its not normal, this totem work on every fear, charm and fear

**Comment by Kejmil** [comment]
http://eu.battle.net/wow/en/forum/topic/1710234564

http://wowwiki.wikia.com/wiki/Timeline_(World_of_Warcraft)

aka you are wrong

**Comment by kadi2** [comment]
http://www.arenajunkies.com/topic/198383-intimidating-shout/

**Comment by Bustyplz** [comment]
My bad, i didn't know there is this difference with the Target fear (stun) and the aoe fear 
but any suggestion for the seduction of the Succubus ?

**Comment by Jenovas** [comment]
The effect the main target gets is not a fear, that is why even 1 DMG Taken breaks the effect.
Targets that are around the warrior and are not targeted by the warrior get a fear effect that breaks after a larger amount of DMG is dealt.
